### ISSUE TITLE
The more PIDs I use, the worse the results become

### ISSUE THREAD

**Issue by stk-code** [KaiyangZhou/deep-person-reid#468]
I have built a training set, that uses about 250.000 PIDs. At least with the video dataset, the loss does not decrease during the training. I assume, that this might have something to do with the number of classes, which will also become 250.000. This feels very huge, especially as output of the model, which would be the shape of [32, 250000] (for a batch size of 32).

Can I reduce the number of classes and keep the training data, or is this just how the training works?
